test/MinimalDelegation.execute.invariant.t.sol

```diff
@@ -130,8 +130,13 @@ contract MinimalDelegationExecuteInvariantHandler is ExecuteFixtures, FunctionCa
         (uint256 nonce,) = _buildNextValidNonce(nonceKey);
         Call[] memory calls = handlerCalls.toCalls();
 
-        SignedCalls memory signedCalls =
-            SignedCallBuilder.init().withCalls(calls).withNonce(nonce).withKeyHash(currentKeyHash);
+        // TODO: remove these once we can test for them, right now we rely on reverts for assertions
+        bool shouldRevert = false;
```

Comment:
why would this be false then? shouldRevert set to true means that it will stop execution when it encounters a revert

Reply:
ohhh im confused I thought it was `canRevert`, where if it was true then it would allow it to